### PR TITLE
Merge feature/game_logic into development

### PR DESCRIPTION
**Pull request by Joseph-Burke**
The authors request to merge the feature/game_logic branch into the development branch having now completed the following work:

-  All game logic has been implemented, allowing the user to play a game of Tic, Tac, Toe with another human player.
-  All Rubocop offences have been addressed and the code now throws no errors.

### CODE REVIEW

**Review by davidmrtz-dev — CHANGES_REQUESTED** [review]
## Hi @Godswilly , @Joseph-Burke  .
### Very good job so far.

However, there are some issues that need to be addressed before move on to the next project.  👨‍💻  ⬇️ 

****

### **Please make the changes required and submit for re-review once you are ready!** 

****

### Status: Changes required. ♻️  
****

Happy coding!  💻  
Have a great day 🙂

If you need more clarifications kindly reach me on slack.
**[TSE:  David Elí]** 
Slack: @ David Eli Martinez Garcia

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+require_relative 'board'
```
Comment (davidmrtz-dev) [inline lib/game.rb:1]: When a draw game occurs, the game does not show that the game is a draw, it just ends its execution.
Kindly add a message to notify the players that a draw occurs.
Optional:
You can add the necessary logic of asking players to start another game.
Screen Shot 2020-08-08 at 1 46 42 PM

**Comment by davidmrtz-dev** [comment]
Please resolve the conflicts between branches to be able to execute the lints in the pull request.

**Comment by Joseph-Burke** [comment]
Additional changes have now been made as requested by @davitomix.

- The mistake in the logic which meant the game.announce_draw method was not called has now been fixed. Now, a draw is announced in the terminal if the game is concluded without a winner.
- The development branch has been merged into the feature/game_logic branch. Conflicts have been resolved and GitHub Actions is now able to run Rubocop, with no offences detected.
- The optional task of allowing the player to restart the game will be addressed in the feature/readme_game_instructions branch when we consider "finishing touches" for the project.

Screenshot 2020-08-09 at 12 40 06

**Comment by lekegitrepo** [comment]
#### Hi @Joseph-Burke and @Godswilly :clap: :clap:
#### You have done an excellent work :100:  by implementing the required changes.
#### This project is approved :rocket:

#### _Reviewed by Temitayo Adeleke_.

**Review by lekegitrepo — APPROVED** [review]
#### Status: Approved

**Pull request merged**